index.html

```diff
@@ -0,0 +1,114 @@
+                sheets containing  Lorem Ipsum passages, and more recently with desktop publishing software like Aldus
+                PageMaker including versions of Lorem Ipsum. </p>
+        </article>
+
+        <!--Second Article-->
+        <article class="article2">
+            <div class="article-title "> Our focus </div>
+            <ul class="article2-list">
+                <li class="article-body-item ">
+                    Interactive
+                </li>
+                <li class="article-body-item">
+                    Mobile
+                </li>
+                <li class="article-body-item">
+                    Technology
+                </li>
+            </ul>
+            <p class="download">
+                <a href="#">Click here to download a pdf for our recent work </a>
+            </p>
+        </article>
+
+        <!--News-->
+        <div class="news">
+            <div class="title ">
+                News
+            </div>
+            
+                
+                <p><a href="#" id="show">Show/Hide news items</a></p>
+                <div class="content" id="mybutton">
+                <p id="drop1">specimetially unchanged. It was popularised in the 1960s with the release of Letraset
+                sheets containing  Lorem Ipsum passages, and more recently with desktop publishing software like Aldus
+                PageMaker including versions of Lorem</p>
+                <p id="drop2">specimetially unchanged. It was popularised in the 1960s with the release of Letraset
+                sheets containing  Lorem Ipsum passages, and more recently with desktop publishing software like Aldus
+                PageMaker including versions of Lorem</p>
+                </div>
+            
+            
+        </div>
+        <!--More Information -->
+
+        <div class="Information">
+            <h1 class="title ">
+                More information
+            </h1>
+            <p class="body">
+                For more information about our product and services, Please contact us the form below.
+            </p>
+            <p class="form-body">Items marked * are required fields</p>
+
+            <form>
+            <label>Conatct name *.</label><br>
+            <input type="input-text" id="username" value="" placeholder="Enter Your name" ><br>
+            <label>Conatct email address *.</label><br>
+            <input type="input-email" id="useremail" value="" placeholder="Enter Your Email"  ><br>
+            <span></span>
+            <label>Conatct phone number *.</label><br>
+            <input type="input-number" id="userphone" value="" placeholder=" Enter phone number"><br>
+            <span></span>
+            <button  id="submit-input" type="submit">Submit</button>
+            </form>
+
+        </div>
+    </div>
+    
+     <script type="text/javascript" src="main.js"></script>
```

Comment:
Because when you put it at the end it means that the javascript will only run in your page once the page is fully loaded.